bin/autosign

```diff
@@ -122,7 +124,8 @@ command :config do |c|
     print.action do |global_options,options,args|
       @logger.debug "print command ran with #{global_options} #{options} #{args}"
       config = Autosign::Config.new({'config_file' => global_options['config']})
-      puts config.settings.to_s
+      require 'yaml'
```

Comment:
This `require` should be moved to the top of the file

Reply:
The require is placed here for lazy loading effect.  If it isn't needed don't load it.